cmake/Platform/Linux/Packaging/snapcraft.yaml.in

```diff
@@ -0,0 +1,64 @@
+name: o3de
+version: '${LY_VERSION_STRING}'
```

Comment:
this is likely actually `${O3DE_INSTALL_DISPLAY_VERSION}` unless it's dev in which case you want `${O3DE_INSTALL_VERSION}` See [Version.cmake](https://github.com/o3de/o3de/blob/0886e0696cda7c1f8520e972026ad004a6f15a7d/cmake/Version.cmake#L91-L94)